mitxpro/settings.py

```diff
@@ -612,7 +613,12 @@
             600,
             description="How many seconds between Hubspot API error checks",
         ),
-    }
+    },
+    # a job is scheduled to run at every-day-at-midnight
+    "generate-course-certificate-daily": {
```

Comment:
Change to just `generate-course-certificates`. The schedule might change in the future so that it's not daily